### DISCUSSION TITLE
Super excited for the next release of copilot

### DISCUSSION THREAD

**Discussion by afgallo** [aws/copilot-cli#2166] [General]
Hi All,

Just leaving a comment here to express my gratitude towards this project. It is pretty amazing!

I am also looking forward to the next release of copilot where it seems that things like rds storage and https redirection might land!

Cheers and keep up the good work 🙏🏻 ❤️

**Comment by efekarakus** [comment]
Thank you so much @afgallo ! The release is now out 😄 https://github.com/aws/copilot-cli/releases/tag/v1.5.0
